ios/RNIterableAPI/RNIterableAPI.m

```diff
@@ -80,7 +80,8 @@ @interface RCT_EXTERN_REMAP_MODULE(RNIterableAPI, ReactIterableAPI, NSObject)
 RCT_EXTERN_METHOD(updateUser: (nonnull NSDictionary *) dataFields
                   mergeNestedObjects: (BOOL) mergeNestedObjects)
 
-RCT_EXTERN_METHOD(updateEmail: (nonnull NSString *) email)
+RCT_EXTERN_METHOD(updateEmail: (nonnull NSString *) email
+                  authToken: (NSString *) authToken)
```

Comment:
Is there a reason setEmail and setUserId are not updated here?

Reply:
Forgot it! Good catch.